### ISSUE TITLE
Audio recording is unreadable/unplayable

### ISSUE THREAD

**Issue by jeshuawoon** [mattermost/mattermost-server#8037]
#### Summary
Tapping on attachments using the mobile web browser version gives me the option to upload a new audio recording, but is uploaded as a file that cannot be played or read when I download it.

#### Steps to reproduce
1. Tap the "attachment" button on mobile web browser version of Mattermost.
2. Select "Recorder" option

![904b5854-6467-4100-8680-d828dfb4c0dd](https://user-images.githubusercontent.com/35112783/34579659-d7b8b22c-f1c4-11e7-99bb-ce57089bb5b4.jpg)

3. Record audio.
4. Upload/send the audio file. (No longer shows up as an "audio" file)
5. Download the file.

Mattermost Version: 4.4.0 (4.4.2) - Mobile web browser
Database: mysql

Device: Xiaomi Redmi 3s
Browser: Chrome

#### Expected behavior
Either an option to upload a recording shouldn't be there at all, or the uploaded recording should appear as a playable audio file.

#### Observed behavior (that appears unintentional) 
Audio file no longer has a format attached to it, and is unreadable/unplayable.
![cap2](https://user-images.githubusercontent.com/35112783/34579850-a978b8d4-f1c5-11e7-9486-1e3f569ec3e9.PNG)

![cap3](https://user-images.githubusercontent.com/35112783/34579907-e2123e68-f1c5-11e7-8702-29631e2d436b.PNG)

#### Possible fixes
Do not allow "Recorder" to launch and record audio files to be uploaded

**Comment by lindy65** [comment]
Hi @jeshuawoon,

Thanks for your feedback,

I tried to reproduce your issue on our [nightly build server](https://pre-release.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676) running off the latest master and it seems there have been some changes since v4.4.  

The only options I see to upload are "camera", "video camera" and "files".  There is no audio available to me.  I tested on Android.
